src/test/java/com/wikia/webdriver/testcases/discussions/CreatingPostTests.java

```diff
@@ -130,22 +140,22 @@ public void userOnDesktopCannotSavePostWithoutCategoryAndDescription() {
   @Test(groups = "discussions-loggedInUsersDesktopPosting")
   @Execute(asUser = User.USER)
   @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
-  @RelatedIssue(issueID = "SOC-3267")
-  public void userOnDesktopCanAddPostWithoutTitle() {
-    final String description = TextGenerator.createUniqueText();
+  public void userOnDesktopCanAddPostWithoutTitle() throws MalformedURLException {
+    String description = TextGenerator.createUniqueText();
 
     PostsListPage page = new PostsListPage().open();
     PostsCreator postsCreator = page.getPostsCreatorDesktop();
 
     final CategoryPill categoryPill = fillPostCategoryWith(postsCreator, description);
 
-    postsCreator.clickSubmitButton();
+    description = addLinkToDescription(postsCreator, description);
+    Assertion.assertTrue(postsCreator.hasOpenGraph(), OPEN_GRAPH_SHOULD_LOAD_MESSAGE);
```

Comment:
I strongly believe that content of `userOnMobileCanAddPostWithoutTitle` should be narrowed down to just that - checking whether user on mobile version can add posts without title. Please consider moving `opengraph` related actions into separate test cases.